app/utils/unsafeCommon.js

```diff
@@ -0,0 +1,17 @@
+// eslint-disable-next-line import/prefer-default-export
+export function removeEmptyObject(value) {
+    if (!isObject(value)) {
+        return value;
+    }
+
+    const newValue = mapToMap(
+        value,
+        (key) => key,
+        (value) => removeEmptyObject(value),
+    );
+    const definedValues = Object.values(newValue).filter(isDefined);
+    if (definedValues.length <= 0) {
+        return undefined;
+    }
+    return newValue;
```

Comment:
@tnagorra sir isObject and mapToMap, where do these come from?
Wild guess, but looks like you didn't run the code before submitting PR? :yum: